### PR TITLE
add input and image component

### PR DESCRIPTION
**Pull request by govindappaarun**
- added basic input component 
 - added image component

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Looks good

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Everything looks good

File: index.html [hunk 1]
```diff
@@ -18,7 +18,6 @@
   </head>
   <body>
     <div class="container">
-               
    <section id="alert">
     <h2>Alert</h2>
     <div class="description">An alert is a simple colored block meant to draw the attention to the user about something. As such, it can be used as a pinned alert in the corner of the viewport. And it has close button to dimiss it.</div>
```
Comment (hsnice16) [inline index.html]: Can't we use `<p>` for description? As It will make it more semantic.

File: index.html [hunk 2]
```diff
@@ -191,6 +190,30 @@ <h2 class="card-title">Our Changing Planet</h2>
           </div>
         </div>
       </div>
+      <h2>Inputs</h2>
```
Comment (hsnice16) [inline index.html]: This can be wrapped into a `<section>`. As It represents a new section for `Inputs`.

File: src/components/input/index.css [hunk 3]
```diff
@@ -7,3 +7,51 @@ input with validation
...
 
 */
+.input-wrapper {
+  display: inline-flex;
+  flex-direction: column;
+  margin: 0.5rem;
+  position: relative;
+  gap: 0.4rem;
+}
+.input-wrapper .error-msg {
```
Comment (hsnice16) [inline src/components/input/index.css:17]: I think we should give at least a line space between two declarations. As it increases readability.

**Review by hsnice16 — CHANGES_REQUESTED** [review]

**Review by ParthParolekar — APPROVED** [review]
Everything looks good to me!

File: style.css [hunk 4]
```diff
@@ -21,3 +21,9 @@
   opacity: 1;
   transition: opacity 0.2s;
 }
+.img-wrapper {
+  width: 40vw;
+}
+.img-responsive {
+  height: 200px;
```
Comment (animeshgarai09) [inline style.css:28]: Not sure why this height property is mentioned here. if this is for documentation purpose I would suggest using a different class name for height. img-responsive class is already mentioned at the top of the file.

**Review by animeshgarai09 — COMMENTED** [review]
Rest look good to me

File: index.html (hunk 2 above)
Comment (govindappaarun) [reply index.html]: fixed, thanks

File: src/components/input/index.css (hunk 3 above)
Comment (govindappaarun) [reply src/components/input/index.css:17]: good to have, so will skip for now

File: style.css (hunk 4 above)
Comment (govindappaarun) [reply style.css:28]: fixed

**Pull request merged**